pfrl/experiments/evaluator.py

```diff
@@ -108,6 +116,10 @@ def _hrl_run_episodes(
 
         a = agent.act_low_level(obs, sg)
         obs_dict, r, done, info = env.step(a)
+
+        if step_number is not None:
+            video_recorder.capture_frame()
+
```

Comment:
if the video_recorder is not enabled, I think we can also delete this `if`s

Reply:
oh good catch.